src/meta/processors/parts/ListHostsProcessor.cpp

```diff
@@ -133,10 +136,13 @@ nebula::cpp2::ErrorCode ListHostsProcessor::allHostsWithStatus(cpp2::HostRole ro
     }
 
     if (now - info.lastHBTimeInMilliSec_ < FLAGS_removed_threshold_sec * 1000) {
+      int64_t expiredTime = FLAGS_heartbeat_interval_secs * FLAGS_expired_time_factor * 1000;
+      if (info.role_ == cpp2::HostRole::AGENT) {
```

Comment:
Maybe use if else to control  FLAGS_heartbeat_interval_secs/FLAGS_agent_heartbeat_interval_secs  and calculate once is more clear?

Reply:
I'll add some comments